src/m365/spfx/commands/project/project-upgrade/upgrade-1.16.1.ts

```diff
@@ -0,0 +1,57 @@
+import { FN001028_DEP_microsoft_sp_list_subscription } from "./rules/FN001028_DEP_microsoft_sp_list_subscription";
+import { FN001029_DEP_microsoft_sp_loader } from "./rules/FN001029_DEP_microsoft_sp_loader";
+import { FN001030_DEP_microsoft_sp_module_interfaces } from "./rules/FN001030_DEP_microsoft_sp_module_interfaces";
+import { FN001031_DEP_microsoft_sp_odata_types } from "./rules/FN001031_DEP_microsoft_sp_odata_types";
+import { FN001032_DEP_microsoft_sp_page_context } from "./rules/FN001032_DEP_microsoft_sp_page_context";
+import { FN001034_DEP_microsoft_sp_adaptive_card_extension_base } from "./rules/FN001034_DEP_microsoft_sp_adaptive_card_extension_base";
+import { FN002001_DEVDEP_microsoft_sp_build_web } from "./rules/FN002001_DEVDEP_microsoft_sp_build_web";
+import { FN002002_DEVDEP_microsoft_sp_module_interfaces } from "./rules/FN002002_DEVDEP_microsoft_sp_module_interfaces";
+import { FN002022_DEVDEP_microsoft_eslint_plugin_spfx } from "./rules/FN002022_DEVDEP_microsoft_eslint_plugin_spfx";
+import { FN002023_DEVDEP_microsoft_eslint_config_spfx } from "./rules/FN002023_DEVDEP_microsoft_eslint_config_spfx";
+import { FN010001_YORC_version } from "./rules/FN010001_YORC_version";
+import { FN015009_FILE_config_sass_json } from "./rules/FN015009_FILE_config_sass_json";
+
+module.exports = [
+  new FN001001_DEP_microsoft_sp_core_library('1.16.1'),
+  new FN001002_DEP_microsoft_sp_lodash_subset('1.16.1'),
+  new FN001003_DEP_microsoft_sp_office_ui_fabric_core('1.16.1'),
+  new FN001004_DEP_microsoft_sp_webpart_base('1.16.1'),
+  new FN001011_DEP_microsoft_sp_dialog('1.16.1'),
+  new FN001012_DEP_microsoft_sp_application_base('1.16.1'),
+  new FN001014_DEP_microsoft_sp_listview_extensibility('1.16.1'),
+  new FN001021_DEP_microsoft_sp_property_pane('1.16.1'),
+  new FN001023_DEP_microsoft_sp_component_base('1.16.1'),
+  new FN001024_DEP_microsoft_sp_diagnostics('1.16.1'),
+  new FN001025_DEP_microsoft_sp_dynamic_data('1.16.1'),
+  new FN001026_DEP_microsoft_sp_extension_base('1.16.1'),
+  new FN001027_DEP_microsoft_sp_http('1.16.1'),
+  new FN001028_DEP_microsoft_sp_list_subscription('1.16.1'),
+  new FN001029_DEP_microsoft_sp_loader('1.16.1'),
+  new FN001030_DEP_microsoft_sp_module_interfaces('1.16.1'),
+  new FN001031_DEP_microsoft_sp_odata_types('1.16.1'),
+  new FN001032_DEP_microsoft_sp_page_context('1.16.1'),
+  new FN001013_DEP_microsoft_decorators('1.16.1'),
+  new FN001034_DEP_microsoft_sp_adaptive_card_extension_base('1.16.1'),
+  new FN002022_DEVDEP_microsoft_eslint_plugin_spfx('1.16.1'),
+  new FN002023_DEVDEP_microsoft_eslint_config_spfx('1.16.1'),
+  new FN002001_DEVDEP_microsoft_sp_build_web('1.16.1'),
+  new FN002002_DEVDEP_microsoft_sp_module_interfaces('1.16.1'),
+  new FN010001_YORC_version('1.16.1'),
+  new FN015009_FILE_config_sass_json(true, `{
+  "$schema": "https://developer.microsoft.com/json-schemas/core-build/sass.schema.json"
+}`)
```

Comment:
Recently tried your PR to upgrade a 1.16.0 project to 1.16.1 and I got some problems at this step. The way I execute the command to get a upgrade report is like this `m365 spfx project upgrade  --output md --shell powershell > "upgrade-report.md"`. I'm more a PowerShell man than Bash 😄. 

When I want to create the new sass.json file I get the option to execute this:

``` PowerShell
@"
{
  "$schema": "https://developer.microsoft.com/json-schemas/core-build/sass.schema.json"
}
"@ | Out-File -FilePath "config\sass.json"
``` 

With running this, the file content isn't properly generated. It's missing the `$schema` key and some invisble chars are added before the first `{`. I haven't done the research yet why this is happening but its just an FYI 😄

Reply:
Good catch! Let's log this as a separate issue as it's not related to this PR but rather a fault with our powershell output